meta-lmp-base/dynamic-layers/clang-layer/recipes-devtools/clang/llvm-project-source.bbappend

```diff
@@ -0,0 +1,3 @@
+FILESEXTRAPATHS:prepend := "${THISDIR}/files:"
+
+SRC_URI:append = " file://Add-missing-cstdint.patch "
```

Comment:
This one can also be sent to meta-clang kirkstone as this is a consequence of a change in oe-core.

Reply:
> This one can also be sent to meta-clang kirkstone as this is a consequence of a change in oe-core.

@ricardosalveti added Upstream-Status, scheduled upstreaming.